releases/0.9.0-incubating/INSTALL.md

```diff
@@ -64,6 +64,20 @@ Download the [signature](https://dist.apache.org/repos/dist/dev/incubator/openwh
 gpg --verify openwhisk-0.9.0-incubating-sources.tar.gz.asc openwhisk-0.9.0-incubating-sources.tar.gz
 ```
 
+You should receive the output messages similar to the following:
```

Comment:
+You should receive the output messages similar to the following:  -> 
 +You should receive output messages similar to the following: